bugbug/models/tracking.py

```diff
@@ -58,12 +59,14 @@ def __init__(self, lemmatization=False):
             ('union', ColumnTransformer([
                 ('data', DictVectorizer(), 'data'),
 
-                ('title', self.text_vectorizer(), 'title'),
+                ('title', self.text_vectorizer(stop_words='english', min_df=0.01), 'title'),
+
+                ('comments', self.text_vectorizer(stop_words='english', min_df=0.01), 'comments'),
```

Comment:
Did you change these because of Out Of Memory issues? If so, `min_df=0.01` seems a little too small.
Please remove `stop_words`

Reply:
Yes, I used 0.01 because of memory issues. I tried with 0.05 and 0.001 both of which gave me the same errors. I've set it to 0.001 as you say its too small, but I can't test the model as it would require more than the 8GB of RAM I'm currently having.